src/NHibernate/Dialect/Dialect.cs

```diff
@@ -1337,6 +1337,11 @@ public virtual JoinFragment CreateOuterJoinFragment()
 			return new ANSIJoinFragment();
 		}
 
+		/// <summary>
+		/// Does this dialect support CROSS JOIN?
+		/// </summary>
+		public virtual bool SupportsCrossJoin => true;
```

Comment:
Shouldn't it be false by default? Otherwise it can be breaking change for some user custom dialects

Reply:
I took the same approach Hibernate did with [getCrossJoinSeparator](https://github.com/hibernate/hibernate-orm/blob/master/hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java#L2368) which enables cross join by default. If you think that is better to have it disabled by default I will do it, otherwise we can add it as a possible breaking change.

Reply:
I'm fine with 
>  add it as a possible breaking change.